oaknational/Oak-Web-Application
# Fix subjects only show active units

## Description

- show only active units on subject listing page

## Issue(s)

Fixes #1162 

## How to test

1. Go to subject listing page
2. Click on _______
3. You should see. KS3 Art and design it would be 3 units and 8 lessons

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
